README.md

```diff
@@ -6,23 +6,27 @@ This repo contains the SD Card contents for all supported EdgeTX Radios.
 
 The contents of this repository are organised by color type and screen size, with zip archives generated for each color type and screen size.
 
-- **c480x272.zip** (480x272 landscape orientation color screen radios)
+- **c480x272.zip** (480x272 pixel landscape orientation color screen radios)
     - TX16s / TX16s mkII
     - T16
     - T18
     - Horus x10s
     - Horus x12s
-    - Generallly, most color screen radios
-- **c320x480.zip** (320x480 portrait orientation color screen radios)
-    - Flysky Nirvana NV14
-- **bw128x64.zip** (128x64 black and white screen radios)
+    - Generally, Most color screen radios...

```

Comment:
This line wasn't didn't get pulled across properly on your merge

Reply:
Thanks, fixed!